docs/source/development/traffic_ops.rst

```diff
@@ -161,7 +161,7 @@ Traffic Ops Project Tree Overview
 		- config/ - Defines configuration structures and methods for reading them in from files
 		- dbhelpers/ - Assorted utilities that provide functionality for common database tasks, e.g. "Get a user by email"
 		- plugin/ - The Traffic Ops plugin system, with examples
-		- riaksvc/ - In addition to handling routes that deal with storing secrets in or retrieving secrets from Traffic Vault, this package provides a library of functions for interacting with Traffic Vault for other handlers to use.
+		- trafficvault/ - This package provides the Traffic Vault interface and associated backend implementations for other handlers to interact with Traffic Vault.
```

Comment:
If we're going to have a package name `trafficvault`, the contents of the `vault` package should be moved there, too.

Reply:
Yeah, I was on the fence with that because `trafficvault` doesn't contain any route handling code and is purely just the integrations w/ the Traffic Vault datastore (currently just Riak), so it would be nice to keep it that way. I was also considering deprecating that "get bucket keys" route altogether because it is basically just a passthrough to Riak, and that isn't really necessary IMO and doesn't really make sense for other backends. I will propose that to the list.